driver/src/main/scala/api/DBMetaCommands.scala

```diff
@@ -127,4 +128,11 @@ trait DBMetaCommands { self: DB =>
       self, command, ReadPreference.primary
     ).map(_ => {})
   }
+
+  /**
+   * Returns 1.0 when the server is responding to commands.
```

Comment:
"if the current primary is responding" ... or need not to used static read preference